src/platform/assets/composables/useAssetBrowserDialog.ts

```diff
@@ -17,30 +19,54 @@ interface AssetBrowserDialogProps {
   onAssetSelected?: (filename: string) => void
 }
 
+interface BrowseOptions {
+  /** Asset type tag to filter by (e.g., 'models') */
+  assetType: string
+  /** Custom modal title (optional) */
+  title?: string
+  /** Called when asset selected */
+  onAssetSelected?: (asset: AssetItem) => void
+}
+
+const PROPS: DialogComponentProps = {
```

Comment:
Why is it yelling at me?

(PROPS is a little generic, not the most searchable constant)